pkg/preparer/listener.go

```diff
@@ -81,23 +80,6 @@ func (l *HookListener) Sync(quit <-chan struct{}, errCh chan<- error) {
 
 			hookPod := pods.NewPod(result.Manifest.ID(), path.Join(l.DestinationDir, event, result.Manifest.ID()))
 
-			// Hooks may specify a user to run as - by default they will run as their pod ID.
-			topLevelConfig := config.LoadFromUnpacked(result.Manifest.Config)
-			hookConfig, err := topLevelConfig.ReadMap("hook")
-			if err != nil {
-				sub.WithField("err", err).Errorln("Hook configuration is invalid")
-				break
-			}
-			runAs, err := hookConfig.ReadString("run_as")
-			if err != nil {
-				sub.WithField("err", err).Errorln("Hook configuration is invalid")
-				break
-			}
-			if runAs == "" {
-				runAs = hookPod.Id
-			}
-			hookPod.RunAs = runAs
```

Comment:
So `manifest.RunAs()` will work for hooks too? Let's verify that's the case, nice improvement.